requirements-dev-minimal.txt

```
@@ -2,10 +2,14 @@ behave>=1.2.6
 coverage
 flake8
 ipdb
+moto>=1.3.15
+mypy
 pre-commit
 pyhamcrest
 pytest
 requirements-tools
-mock
-moto>=1.3.15
-mypy
+types-cachetools==5.2.1
+types-PyYAML==6.0.12
+types-requests==2.28.11
+types-retry==0.9.9
+types-urllib3==1.26.25
```

these should only be pinned in requirements-dev.txt
```suggestion
types-cachetools
types-PyYAML
types-requests
types-retry
types-urllib3
```

---

(this is the only blocking comment)

---

oops, nice catch, some copy pasta 😅